### PR TITLE
added Powergems tag

### PR DESCRIPTION
**Pull request by LunarcatOwO**
Added the powergems tag to everything I think that would be sent to console

### CODE REVIEW

File: src/main/java/me/iseal/powergems/commands/debugCommand.java [hunk 1]
```diff
@@ -17,16 +17,16 @@ public class debugCommand implements CommandExecutor {
     public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label,
             @NotNull String[] args) {
         if (!(sender instanceof Player)) {
-            sender.sendMessage(ChatColor.DARK_RED + "You need to be a player to execute this command!");
+            sender.sendMessage(ChatColor.DARK_RED + SingletonManager.configManager.getGeneralConfigManager().getPluginPrefix() + "You need to be a player to execute this command!");
             return true;
         }
         Player plr = (Player) sender;
         if (!plr.hasPermission(command.getPermission())) {
-            plr.sendMessage(ChatColor.DARK_RED + "You do not have permission to execute this command.");
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/commands/debugCommand.java:25]: Why was the ChatColor removed here?

File: src/main/java/me/iseal/powergems/commands/giveAllGemCommand.java [hunk 2]
```diff
@@ -18,12 +18,12 @@ public class giveAllGemCommand implements CommandExecutor {
     public boolean onCommand(@NotNull CommandSender commandSender, @NotNull Command command, @NotNull String s,
             @NotNull String[] args) {
         if (!(commandSender instanceof Player)) {
-            commandSender.sendMessage("Only players can execute this command!");
+            commandSender.sendMessage(SingletonManager.configManager.getGeneralConfigManager().getPluginPrefix() + "Only players can execute this command!");
             return true;
         }
         Player plr = (Player) commandSender;
         if (!plr.hasPermission(command.getPermission())) {
-            plr.sendMessage(ChatColor.DARK_RED + "You do not have permission to execute this command.");
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/commands/giveAllGemCommand.java:26]: Same as before, why was the chatcolor removed?

**Review by iseal56 — CHANGES_REQUESTED** [review]
Generally good, try to instanciate gcm if not instanciated already though.

**Review by iseal56 — COMMENTED** [review]
Could be pushed, but you really should instantiate GeneralConfigManager every time it is used more than once in a class.

**Comment by LunarcatOwO** [comment]
Alr :/

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**